src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx

```diff
@@ -160,42 +162,93 @@ function WorldWideTelescope({
     });
   }
 
-  function handleDrag(mouse) {
-    if (isDragging) {
+  function getClientXY(interaction) {
+    let position = undefined;
+    if (interaction.type === "touchstart" || interaction.type === "touchmove") {
+      if (!interaction?.touches?.[0]?.clientX || !interaction?.touches?.[0]?.clientY) {
+        return;
+      }
+      if (interaction?.touches?.length > 1) {
+        const touches = interaction.touches;
+        position = [
+          [touches[0].clientX, touches[0].clientY],
+          [touches[1].clientX, touches[1].clientY]
+        ];
+      }
+      else {
+        const touch = interaction.touches[0]; 
+        position = [touch.clientX, touch.clientY];
+      }
+    }
+    else { // mouse
+      position = [interaction.clientX, interaction.clientY];
+    }
+    if (!position) {
```

Comment:
This if-statement seems unnecessary. If `!position` is true, it means that it's undefined, which means that we will return undefined even if we just return position as is